### PR TITLE
Onboarding 3

### PR DESCRIPTION
**Pull request by mjadach-iv**

### CODE REVIEW

File: src/components/Stepper/index.tsx [hunk 1]
```diff
@@ -77,6 +77,7 @@ type StepProps = {
   name: string;
   position: number;
   state: StepState;
+  stepComplated: StepState;
```
Comment (diegoalzate) [inline src/components/Stepper/index.tsx:80]: rename to lastStepCompleted

File: src/pages/staking-hub/onboarding.tsx [hunk 2]
```diff
@@ -36,30 +33,56 @@ function Onboarding() {
...
+
+  function whatIsCurrentStep(page: number) {
+    switch (page) {
+    case 4:
+      return 1;
+
+    default:
+      return 0;
+    }
+  }
```
Comment (diegoalzate) [inline src/pages/staking-hub/onboarding.tsx:55]: this is still confusing. I think I would go for something like an object:

```
const STEPS =  {
 CREATE_SAFE: 1,
...
}
```

File: src/router.tsx [hunk 3]
```diff
@@ -33,7 +33,7 @@ import UpdateNodePage from './steps/updateNode';
 import SetupNodePage from './steps/setupYourNode';
 import AddedToWhitelist from './steps/addedToWhitelist';
 import JoinWaitlistPage from './steps/joinWaitlist';
-import WhatYouWillNeedPage from './steps/whatYouWillNeed';
+import WhatYouWillNeedPage from './steps/step0_whatYouWillNeed';
```
Comment (diegoalzate) [inline src/router.tsx:36]: this comment is not specific to this exact file but the naming convention used in this file goes against what we have used in the rest of the repo so maybe we can have a folder step0 and inside that folder we can have that file

**Review by diegoalzate — COMMENTED** [review]
some comments that we could branch into different prs if neccesary

File: src/router.tsx (hunk 3 above)
Comment (mjadach-iv) [reply src/router.tsx:36]: folder is ok, @diegoalzate Diego please create a task to do on the topic, tbd when all steps are working

File: src/pages/staking-hub/onboarding.tsx (hunk 2 above)
Comment (mjadach-iv) [reply src/pages/staking-hub/onboarding.tsx:55]: Good idea, will do sth like that in next PR

File: src/components/Stepper/index.tsx (hunk 1 above)
Comment (diegoalzate) [reply src/components/Stepper/index.tsx:80]: ok

File: src/router.tsx (hunk 3 above)
Comment (diegoalzate) [reply src/router.tsx:36]: ok

**Review by diegoalzate — APPROVED** [review]

**Pull request merged**